CHANGELOG.md

```
@@ -1,5 +1,6 @@
 # Changelog
-
+### [3.0.8]
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)